src/Router.ts

```diff
@@ -9,12 +9,14 @@ export class Context<Data = any, Params = any> {
   readonly params: Params
   response: Response
   data: Data
+  event: FetchEvent
 
-  constructor(request: Request, response: Response, params: Params, data: Data) {
+  constructor(request: Request, response: Response, params: Params, data: Data, event: FetchEvent) {
```

Comment:
Maybe we should swap out `request` for `event`?

Reply:
Strong agree! This makes a lot of sense.

Reply:
Then it will be 100% breaking and doesn't make much sense to keep `getResponseForRequest`. We okay with that?